packages/documentation-site/patternfly-docs/content/design-guidelines/content/brand-voice-and-tone.md

```diff
@@ -51,33 +57,33 @@ You can’t *tell* your users what your voice is. You need to *show* them who yo
 
 </div>
 
-**Voice trait**: Approachable
+#### Voice trait: Approachable
 
 **Description**: People are comfortable engaging with us. We’re open to listening and changing our ways when better ideas come along.
 
 <div class="ws-content-table">
 
 | **Do** | **Don't** |
 |--------|-----------|
-| Say what you mean.                                             | Don’t use jargon, idioms, bizspeak, or formal language. |
-| Be direct and transparent with descriptive, specific language. ||
+| Say what you mean. | Don’t use jargon, idioms, bizspeak, or formal language. |
+| Be direct and transparent with descriptive, specific language. |
 
 </div>
 
-**Voice trait**: Collaborative
+#### Voice trait: Collaborative
 
 **Description**: We embody Red Hat’s open source mission with our collaborative working style and our sense of community.
 
 <div class="ws-content-table">
 
 | **Do** | **Don't** |
 |--------|-----------|
-| Deliver content in a way that includes everyone.                                     | Don’t rely on inside jokes, colloquial expressions, culture-specific examples, or other alienating language to get your point across. |
-| Use “you” and the active voice to put emphasis on the user and power in their hands. | Don’t use “I” too much. Make the user the star of every story you tell. |
+| Deliver content in a way that includes everyone. | Don’t rely on inside jokes, colloquial expressions, culture-specific examples, or other alienating language to get your point across. |
```

Comment:
I notice that in these tables, for the "Do" column, the sentences don't start with "Do" but with the "Don't" column, all the sentences start with "Don't" - would keep that consistent whichever way you go!